### DISCUSSION TITLE
Index.html reports 0 on everything

### DISCUSSION THREAD

**Discussion by bmackeydhcs** [danielpalme/ReportGenerator#632] [Q&A] [answered]
I have the following report:
![image](https://github.com/danielpalme/ReportGenerator/assets/142840661/b9a65e47-bb17-4e79-b0ca-3cb9855db211)

I consume the report with this command:

`reportgenerator -reports:./MyPath/Me.Tests/coverage/results.json -targetdir:./coverage -reporttypes:Html_Dark`

And the Index.html report contains 0 on everything:
![image](https://github.com/danielpalme/ReportGenerator/assets/142840661/27ab0b30-7127-44a5-9693-580f6004f9c7)

**EXPECTED RESULT:**

At least some non-zero values.  Some pages and assemblies should be shown.

**Comment by bmackeydhcs** [comment, marked as answer]
My report is generated via the command:

`dotnet test MyPath/Me.Tests/ /p:CollectCoverage=true /p:CoverletOutputFormat=lcov /p:CoverletOutput=./coverage/ /p:Exclude=[xunit.*]*.MyPath/My.sln /p:ExcludeByFile=\"**/*.cshtml\"`

I modified my report to be

`/p:CoverletOutputFormat=cobertura`
instead of 
`/p:CoverletOutputFormat=lcov`

This created an XML based report as opposed to a JSON based report.

And then I got proper results from reportgenerator.

**Reply by bmackeydhcs** [reply]
I'm not really sure why this worked?

**Reply by danielpalme** [reply]
It works because ReportGenerator supports the XML format. It does not support the JSON format
